cudapoa/src/allocate_block.hpp

```diff
@@ -381,32 +415,41 @@ class BatchBlock
         size_t mem_per_batch = memory_usage_quota * free; // Using memory_usage_quota of GPU available memory for cudapoa batch.
 
         int64_t sizeof_ScoreT       = 2;
+        int64_t sizeof_TraceT       = use16bitTrace(batch_size) ? 2 : 1;
```

Comment:
good point, since it was a static function, I did not use `sizeof(ScoreT)` or `sizeof(TraceT)`. But changing it to `sizeof(int8_t)`, `sizeof(int16_t)`, and `sizeof(int32_t)` makes the code more readable. done.